src/tests/tablet/schedule/QuantityOfCharactersOfOrganizerIsLimitedTo25.java

```diff
@@ -0,0 +1,48 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.utils.readers.ExcelReader;
+
+/**
```

Comment:
according this title you are verifying the subject text box but according your test case title you are verifying the organizer text box what is right?